fdbserver/DataDistribution.actor.cpp

```diff
@@ -1898,15 +1894,20 @@ ACTOR Future<Void> scheduleBulkLoadJob(Reference<DataDistributor> self, Promise<
 						} else if (task.onAnyPhase({ BulkLoadPhase::Submitted,
 						                             BulkLoadPhase::Triggered,
 						                             BulkLoadPhase::Running })) {
-							// Limit parallelism
-							loop {
-								if (self->bulkLoadParallelismLimitor.canStart()) {
-									break;
+							// Do not monitor any task until all tasks are submitted.
+							// Otherwise, the parallelism limitor will slow down the task submission.
+							if (self->bulkLoadJobManager.get().allTaskSubmitted) {
```

Comment:
So if 250k tasks, we keep submitting and they start running and no parallelism monitoring while submission is ongoing?  This ok?

Reply:
the bulkload task engine is limiting the parallelism.